code/game/jobs/job/civilian.dm

```diff
@@ -216,6 +216,28 @@
 		return TRUE
 
 
+/datum/job/journalist
+	title = "Journalist"
+	flag = JOURNALIST
+	department = "Civilian"
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the head of personnel"
+	selection_color = "#ddbbbb"
+	access = list(access_journalist, access_maint_tunnels, access_sec_doors)

```

Comment:
I am very unsure about the security access.
There should be the expectation that security works with them, but they should not have unchecked access to security.

If you wish to keep certain details about a ongoing situation confidential that is so much harder if the journalist has access to security. (Especially if they are not working under the HoS; Which I dont recommend)

Reply:
They don't get it on the server- check minimal access, I think. Also, the sec door access was a leftover. Planning to remove it anyway.

Reply:
ye, we use minimal access lists primarily.